goagen/utils/files_test.go

```diff
@@ -0,0 +1,84 @@
+package utils
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+)
+
+var sep = string(os.PathSeparator)
+
+// Test RemoveFiles. Creates new directories and files in temp directory.
+// Expects that RemoveFiles delete only files, and leaving directory structure intact.
+// Files from .svn directory should not be deleted.
+func TestRemoveFiles(t *testing.T) {
+
+	//Creating temp directory. When error occures, temp directory would remain, for visual check.
+	testDir := filepath.Join(".", time.Now().Format("20060102150405"))
+	currentDir := testDir
+	createDir(currentDir, t)
+
+	currentDir = testDir + sep + "app"
```

Comment:
you can use `filepath.Join(testDir, "app")` instead of the above which might be a bit more readable.